concatenate_ice/concat_ice_daily.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/bash
+#concatenate sea-ice daily output
+#script inspired from https://github.com/COSIMA/1deg_jra55_ryf/blob/master/sync_data.sh#L87-L108
+
+out_dir=$(ls -td archive/output??? | head -1)/ice/OUTPUT #latest output dir only
+
+for f in $out_dir/iceh.????-??-01.nc; do
+    #concat daily files for this month
+    echo "doing ncrcat -O -L 5 -4 ${f/-01.nc/-??.nc} ${f/-01.nc/-daily.nc}"
```

Comment:
Quoting the variable substitutions will echo something different to the screen than is actually run. Quoted it outputs something like
```
doing ncrcat -O -L 5 -4 iceh.2001-03-??.nc iceh.2001-03-daily.nc
```
but the acutal command will expand out the wildcard. Now this may be exactly what you want (it is more concise), but when trying to debug when something goes awry IMO it is usually good practice to echo what the shell actually thinks is going to happen.

Reply:
Thanks. I had noticed this but hadn't thought about it much!